app/src/components/filterEntities/entityItemStartTime.jsx

```diff
@@ -53,18 +44,15 @@ const presets = [
     value: {
       start: moment()
         .startOf('day')
-        .subtract(1, 'months')
+        .subtract(29, 'days')
```

Comment:
Why 29 days? Months have different day count.

Reply:
In dropdown user sees not 1 month, but `30 days`.